libs/wingsdk/src/cloud/website.ts

```diff
@@ -0,0 +1,95 @@
+import { isAbsolute, resolve } from "path";
```

Comment:
Noticed another issue when working with the website resource on aws:
Need to invalidate cache on CloudFront after every change to the static website's code.
I think as part of `terraform apply` we should have some logic to invalidate some/all of the static files (need to leave an escape hatch for users to decide to remove specific files/patterns only)

Reply:
I cannot find a cdktf module to perform cache invalidation :( 
I wonder if I'm missing something
I could find one for cdk:
https://docs.aws.amazon.com/cdk/api/v1/docs/aws-s3-deployment-readme.html#cloudfront-invalidation

Reply:
and see that it's not trivial in terraform :( 
https://stackoverflow.com/questions/69794727/how-can-i-invalidate-aws-cloudfront-distribution-cache-using-terraform

Reply:
I think we should open an issue and think what can be the best way to achieve it based on the available tools... @staycoolcall911  WDYT?

Reply:
Yes, we can open a separate issue and leave this to a following PR, but then it must be documented, because every user of the Website resource will run into that sooner or later. Perhaps @Chriscbr, @hasanaburayyan or @eladb will have an idea.

Reply:
Ah right, cache invalidation - the hardest problem in computer science 😅 yes I remember now CDK had some dedicated code for handling this for users... it looks like it was written in Python as part of the BucketDeployment resource [here](https://github.com/aws/aws-cdk/blob/c81d115955dbb27ce873ed7c9d71cc0dc8eacf99/packages/aws-cdk-lib/aws-s3-deployment/lib/lambda/index.py#L130-L131)

I feel like we eventually want to handle this using the `OnDeploy` resource (basically, a `cloud.Function` that runs during deployment to call the AWS SDK and perform a cache invalidation with minimal permissions). But the approach mentioned on StackOverflow (creating a local_exec resource) sounds okay as a workaround - documenting as @staycoolcall911 mentioned also works 👍

Reply:
opened an issue: https://github.com/winglang/wing/issues/2404